core/src/main/java/com/novoda/noplayer/internal/exoplayer/NoPlayerAdsLoader.java

```diff
@@ -293,6 +296,17 @@ void disableAdverts() {
         advertsDisabled = true;
     }
 
+    void skipAdverts(@NonNull AdvertBreak advertBreak) {
```

Comment:
I think this just happens to work because the `disable` and `enable` aren't called again in the same playback session. I would expect this to not work if the disable and then the enable were called immediately. I think this is fine, we are kind of backed into a corner as far as the states go.